src/lib/header-bidding/prebid/modules/analyticsAdapter.ts

```diff
@@ -1,73 +1,32 @@
 // see http://prebid.org/dev-docs/integrate-with-the-prebid-analytics-api.html
+import { log } from '@guardian/libs';
 import adapter from 'prebid.js/libraries/analyticsAdapter/AnalyticsAdapter.js';
 import adapterManager from 'prebid.js/src/adapterManager.js';
-import { EVENTS } from 'prebid.js/src/constants.js';
-import * as utils from 'prebid.js/src/utils.js';
 import { ajax } from 'prebid.js/src/ajax.js';
-
-// eslint-disable-next-line prebid/validate-imports
-import { log } from '@guardian/libs';
+import { EVENTS } from 'prebid.js/src/constants.js';
+import type {
+	AnalyticsAdapter,
+	AnalyticsConfig,
+	AnalyticsOptions,
+	BidArgs,
+	EventData,
+	Handler,
+	RequestTemplate,
+} from '../../../../types/prebid';
 
 /*
  * Update whenever you want to make sure you're sending the right version of analytics.
  * This is useful when some browsers are using old code and some new, for example.
  */
 const VERSION = 9;
 
-const analyticsType = 'endpoint';
-const SENDALL_ON = {};
-
-// Look there: http://jsben.ch/qhIE6
-SENDALL_ON[EVENTS.AUCTION_END] = true;
-SENDALL_ON[EVENTS.BID_WON] = true;
-
-let analyticsAdapter = Object.assign(adapter({ analyticsType }), {
```

Comment:
There was a lot of "using things before they were defined" eslint errors so things have moved around a bit.